scipy/sparse/linalg/interface.py

```diff
@@ -823,5 +823,4 @@ def aslinearoperator(A):
                                   rmatmat=rmatmat, dtype=dtype)
 
         else:
-            A = np.atleast_2d(np.asarray(A))
-            return MatrixLinearOperator(A)
+            raise TypeError('type not understood')
```

Comment:
I reset this file to master. There should not be any changes to this file in #9 after this is merged.